src/screens/ShowDetailsScreen.tsx

```diff
@@ -1,13 +1,66 @@
+import { useEffect, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import { getMovieProviders } from '../helpers/getMovieUtils';
+import { MovieProviders, MovieDetailsData } from '../types/tmdb';
+
 /**
  * Screen to show more details of a specific show
  * Rendered after user clicks on show card
  * 
  * @returns {JSX.Element}
  */
 export default function ShowDetailsScreen(): JSX.Element {
+	const [providers, setProviders] = useState<MovieProviders>();
+	const location = useLocation();
+	console.log(location);
+
+	useEffect(() => {
+		const handler = async () => {
+			const data = await getMovieProviders(location.state.details.id);
+			setProviders(data) as unknown as MovieProviders;
+		};
+		handler();
+	}, []);
+
+	// TODO: Possibly create utility function 
+	const ratingHandler = (arr: MovieDetailsData): JSX.Element | null => {
+		for (let i = 0; i < arr.release_dates.results.length; i++) {
+			if (arr.release_dates.results[i].iso_3166_1 === 'US') {
+				return <p>{arr.release_dates.results[i].release_dates[0].certification}</p>;
+			}
+		}
+		return null;
+	};
+
 	return (
-		<>
-			<h1>Show Details Page</h1>
-		</>
+		<section>
+			{providers && (
+				<div className='flex'>
+					<div>
```

Comment:
Does this div serve any purpose?

Reply:
@Thenlie It's just a parent container. IMO it visually looks cleaner and consistent with the rest of with JSX. And easier/simpler to style when we get there.

Reply:
My vote is to remove, but not a blocking issue.